config/templates/kava/master/initstate/.kava/config/genesis.json

```diff
@@ -506,6 +506,10 @@
               "denom": "busd",
               "amount": "10000000000000000"
             },
+            {
+              "denom": "erc20/tether/usdt",
+              "amount": "10000000000000000"
+            },
```

Comment:
to prevent errors & failed invariants, this will require the state changes that also correspond to the token being minted by the contract. those erc20 tokens will need to be owned by the evmutil module account to ensure that these sdk coins are properly backed